### DISCUSSION TITLE
Edit docker-compose from cloned git project

### DISCUSSION THREAD

**Discussion by Donubu** [coollabsio/coolify#4129] [General]
Hello, i created a new resource based on a github project. On build pack (type docker composer) i got the "docker-compose.yaml" and all is ok. If i deploy everything works. I need to add some changes to docker-compose, but the editor is read-only. there is a way to update that file? my intention is add some labels to traffik adding basic auth to prevent access without a password.

**Comment by Donubu** [comment]
:(

**Reply by Procuria** [reply]
AFAIK there is no way to do that. 
I always fork those projects and add the needed adjustments in the fork. 

But if there is another way I'll be also happy to learn about it.

**Reply by Donubu** [reply]
how you can obfuscate the user password encoded from new added labels when push the changes on the forked git?

**Reply by Procuria** [reply]
If I understand your question correctly, you should be already covered by the Base64 obfuscation of passwords, needed in the Taefik label configuration.   
At least that's what I understand reading the first paragraph of the [Traefik basic auth middleware documentation](https://coolify.io/docs/knowledge-base/traefik/basic-auth) by Coolify
